### PR TITLE
toJson fixes

### PR DESCRIPTION
**Pull request by dopey**

### CODE REVIEW

File: step-ca/configuration.mdx [hunk 1]
```diff
@@ -1544,7 +1544,7 @@ a root certificate.
 
 The default template for an intermediate certificate is:
 
-```
+```json
```
Comment (maraino) [inline step-ca/configuration.mdx:1547]: Is the type json properly rendered? The template itself is not json.

**Review by maraino — APPROVED** [review]
LGTM, I've added a small comment to verify that `json` format shows the templates without red blocks.

File: step-ca/configuration.mdx (hunk 1 above)
Comment (dopey) [reply step-ca/configuration.mdx:1547]: That's what all the other templates are using and it seems to render just fine.

Comment (maraino) [reply step-ca/configuration.mdx:1547]: 👍

**Pull request merged**